.github/label-actions.yml

```diff
@@ -0,0 +1,34 @@
+# When `devnet-e2e-test` is added, also assign `devnet` to the PR.
+devnet-e2e-test:
```

Comment:
Please make sure we have good `label` comments for all these github labels as well

Reply:
I added a note about increased CI times, though! Thanks for that idea.
Perhaps we [can extend](https://github.com/dessant/label-actions/blob/ade7bcd4c1b30de6ba8e556cc31301fd4f79ca65/src/index.js#L95) this github action to allow more templating functionality - that way, we could create a message with links to grafana or something similar that points to the network specific page. Anyway, I'd say it is an overkill for now :)